rudder-web/src/main/scala/com/normation/rudder/web/snippet/HomePage.scala

```diff
@@ -283,6 +288,19 @@ class HomePage extends Loggable {
   }
 
   def inventoryInfo() = {
+
+
+    val osTypes = AixOS       ::
+      SolarisOS               ::
+      BsdType.allKnownTypes   :::
+      LinuxType.allKnownTypes :::
+      WindowsType.allKnownTypes
+
+    // mapping between os name and their string representation (printed on screen).
+
+    val osNames = JsObj(osTypes.map {os =>
+      (S.?("os.name." + os.name), Str(os.name))}:_*)
```

Comment:
Put it on one line or do 
<pre>
JsObj(osTypes.map {
            os => (S.?("os.name." + os.name), Str(os.name))}:_*)
          }:_*)
                  
</pre>